skimage/filters/tests/test_unsharp_mask.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from skimage.filters import unsharp_mask
+import pytest
+
+
+@pytest.mark.parametrize("shape,multichannel",
+                         [((101,), False),
```

Comment:
This is a huge testing matrix. Could it be less extensive so that tests run faster?

Reply:
Although the coverage is near to perfect here, I agree with @emmanuelle that it is better to decrease the number of tests. The main reason for this is the limitation of our free Travis CI plan: we are limited in execution time of the test suite, and having 20sec for a single function might be a bit excessive.

Reply:
I reduced the test sizes significantly, now it should run in a few secs max.
If it is still considered slow, I could tune it a bit more.